README.md

```diff
@@ -99,7 +99,7 @@ loginButton.addLoginListener(new LoginListener() {
 ```
 
 
-For more information, refer to the [LINE SDK for Android guide](https://developers.line.biz/en/docs/android-sdk/) on the [LINE Developers site](https://developers.line.biz).
+For more information, refer to the [LINE SDK for the Android guide](https://developers.line.biz/en/docs/android-sdk/) on the [LINE Developers site](https://developers.line.biz).
```

Comment:
```suggestion
For more information, refer to the [LINE SDK for Android guide](https://developers.line.biz/en/docs/android-sdk/) on the [LINE Developers site](https://developers.line.biz).
```

Let's leave as-is. the product is "LINE SDK for Android", so it's right to say `the [LINE SDK for Android guide]`